save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt

```diff
@@ -252,32 +252,29 @@ class LnkUserOrganizationController(
         lnkUserOrganizationService.getSuperOrganizationsWithRole((authentication.details as AuthenticationDetails).id)
     )
 
-    @GetMapping("/by-user")
+    @PostMapping("/by-user-and-filters")
     @RequiresAuthorizationSourceHeader
     @PreAuthorize("permitAll()")
     @Operation(
-        method = "GET",
-        summary = "Get user's organizations by status",
-        description = "Get organizations by status available for the current user.",
+        method = "POST",
+        summary = "Get user's organizations by filters",
```

Comment:
Google Translate suggests this is rephrased as

```suggestion
        summary = "Get the list of organizations available to the current user and matching the filters, if any",
```

Reply:
Done